app/features/settings/views/main-view.tsx

```diff
@@ -0,0 +1,102 @@
+import { Edit, Landmark, QrCode, Share, X } from 'lucide-react';
+import { Button } from '~/components/ui/button';
+import {
+  SidebarContent,
+  SidebarFooter,
+  SidebarGroup,
+  SidebarHeader,
+  SidebarSeparator,
+  useSidebar,
+} from '~/components/ui/sidebar';
+import { formatUnit } from '~/lib/formatting';
+import { canShare, shareContent } from '~/lib/share';
+import { SettingsNavButton } from '../components/settings-nav-button';
+import { useSettingsSidebar } from '../settings-sidebar-provider';
+
+const username = 'satoshi';
+const activeMintName = 'Coinos';
+const activeMintBalance = 10_000;
+const activeMintUnit = 'usd';
+
+export function MainView() {
+  const { setOpen, setOpenMobile, isMobile } = useSidebar();
+  const { navigateToView } = useSettingsSidebar();
+
+  const handleShare = async () => {
+    const data = {
+      title: `${username}'s Boardwalk`,
+      url: `https://boardwalkcash.com/${username}`,
+    };
+    await shareContent(data);
+  };
+
+  return (
+    <>
+      <SidebarHeader>
+        <div className="flex items-center justify-between">
+          <Button
+            variant="ghost"
+            size="icon"
+            onClick={() => (isMobile ? setOpenMobile(false) : setOpen(false))}
+          >
+            <X />
```

Comment:
imo it's kind of odd that we have that buton for opening sidebar still present once opened and then you can close the sidebar with that button or by clicking this x but perhaps I should not be focusing on stuff like that atm as we will do another ui iteration?

Reply:
Yea I didn't want to spend time on that. I think we want the Settings icon to stay fixed where it is and have the sidebar open on top of it, then only close with the X. Moving the icon when the sidebar opens is the default behavior and I couldn't figure out how to change it. Shouldn't be hard to modify the component though